### ISSUE TITLE
Parameter missmatch ?

### ISSUE THREAD

**Issue by daslicht** [not-only-code/qtranslate-slug#115]
Here : 
https://github.com/not-only-code/qtranslate-slug/blob/trunk/includes/class-qtranslate-slug.php#L2408
we have the function call : 

```
qts_insert_term_input('name', __('Name','qts'), $termname, $language,"edit");
```

but : 

```
private function qts_insert_term_input($id,$name, $language,$action){
```

only accepts 4 parameters ?

**Comment by LC43** [comment]
Hi @daslicht , thanks for reporting. hm, i need to review that.

**Comment by daslicht** [comment]
see also my post at the wordpress support forum, I get some errors

**Comment by daslicht** [comment]
see: https://wordpress.org/support/topic/creating-categories-not-working-reliable

**Comment by LC43** [comment]
fixed in 1.1.18, thanks @daslicht !!

**Comment by daslicht** [comment]
You are welcome !
